frontend/src/metabase/visualizations/visualizations/SmartScalar/SmartScalar.jsx

```diff
@@ -199,7 +196,23 @@ function PreviousValueComparison({
           width: getChangeWidth(width),
         })
       : display.percentChange;
-  const separator = <Separator> • </Separator>;
+
+  const separatorColor = isNightMode
+    ? lighten(theme.fn.themeColor("text-medium"), 0.15)
+    : lighten(theme.fn.themeColor("text-light"), 0.25);
```

Comment:
To investigate if we can use Mantine's shades system to handle this, such as `text-medium.3` or `text-light.9`. If it maps to the 10-color shade then we can remove this!